src/components/widgets/NavigationTabsWidget/NavigationTabsWidget.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * TESLER-UI
+ * Copyright (C) 2018-2020 Tesler Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react'
+import NavigationTabs from '../../ui/NavigationTabs/NavigationTabs'
+import { NavigationWidgetMeta } from '../../../interfaces/widget'
+
+export interface NavigationTabsWidgetProps {
+    meta: NavigationWidgetMeta
```

Comment:
Why the entire meta object is a prop when only `navigationLevel` field is actually used?